associations/models/news.py

```diff
@@ -7,17 +7,18 @@
 class News(models.Model):
     id = models.AutoField(primary_key=True)
 
-    title = models.CharField(max_length=200, blank=True, null=True, default=None)
+    association = models.ForeignKey(Association, on_delete=models.CASCADE)
+    author = models.ForeignKey(User, on_delete=models.CASCADE)
 
     date = models.DateTimeField(auto_now=True)
-    author = models.ForeignKey(User, on_delete=models.CASCADE)
 
-    association = models.ForeignKey(Association, on_delete=models.CASCADE)
+    title = models.CharField(max_length=200, blank=True, null=True, default=None)
```

Comment:
Maybe blank title shouldn't be authorized

Reply:
Indeed.